audio/speech/getting-started/get_started_with_chirp_3.ipynb

```diff
@@ -336,6 +432,73 @@
       "source": [
         "display(Audio(response.audio_content))"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LYKj5gmDG7nT"
+      },
+      "source": [
+        "### Synthesize speech using streaming processing\n",
+        "\n",
+        "Chirp 3 HD voices also support streaming text-to-speech conversion using the `streaming_synthesize` method.  Unlike the standard `synthesize_speech`, which handles single requests, `streaming_synthesize` processes continuous streams of text, generating corresponding audio streams.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ll9rWo-dQ8Cx"
+      },
+      "outputs": [],
+      "source": [
+        "def synthesize_streaming(\n",
+        "    text_iterator,\n",
+        "):\n",
+        "    \"\"\"Synthesizes speech from an iterator of text inputs and yields audio content as an iterator.\n",
+        "\n",
+        "    This function demonstrates how to use the Google Cloud Text-to-Speech API\n",
+        "    to synthesize speech from a stream of text inputs provided by an iterator.\n",
+        "    It yields the audio content from each response as an iterator of bytes.\n",
+        "\n",
+        "    \"\"\"\n",
+        "    client = texttospeech.TextToSpeechClient()\n",
+        "\n",
+        "    streaming_config = texttospeech.StreamingSynthesizeConfig(\n",
+        "        voice=texttospeech.VoiceSelectionParams(\n",
+        "            name=\"en-US-Chirp3-HD-Charon\", language_code=\"en-US\"\n",
```

Comment:
Should this use the inputs previously selected by the user?

Reply:
good catch!

Reply:
done